packages/subgraph/tests/helpers/method-classes.ts

```diff
@@ -536,12 +543,13 @@ class DaoMethods extends Dao {
 // Token Voting
 class TokenVotingVoterMethods extends TokenVotingVoter {
   withDefaultValues(): TokenVotingVoterMethods {
-    this.id = Address.fromHexString(CONTRACT_ADDRESS)
-      .toHexString()
-      .concat('_')
-      .concat(ADDRESS_ONE);
-    this.address = ADDRESS_ONE;
-    this.plugin = Address.fromHexString(CONTRACT_ADDRESS).toHexString();
+    const memberAddress = Address.fromString(ADDRESS_ONE);
+    const memberId = generateEntityIdFromAddress(memberAddress);
```

Comment:
double check, if you don't the above case (https://github.com/aragon/osx/pull/520#discussion_r1449527434) here too